jobs/scripts/addon-tests.sh

```diff
@@ -1,14 +1,13 @@
 #!/bin/sh
-if [ "{plone-version}" = "4.3" ]; then
-    python bootstrap.py -c jenkins.cfg
-    bin/buildout buildout:git-clone-depth=1 -c jenkins.cfg
-else
-    python bootstrap.py --setuptools-version 33.1.1 --buildout-version 2.8.0 -c core.cfg
-    bin/buildout buildout:git-clone-depth=1 -c core.cfg
-fi
+python bootstrap.py
```

Comment:
The conditional needs to stay as this is (un)used for 4.3/5.0/5.1/5.2 jobs.

The ``(un)`` is due to that nobody is actually using it :sweat_smile: